Makefile

```diff
@@ -23,6 +23,7 @@ reset-db:
 	@sudo -iu postgres bash -c "psql -c 'DROP DATABASE IF EXISTS coobs;'"
 	@sudo -iu postgres bash -c "psql -c 'CREATE DATABASE coobs;'"
 	@sudo -iu postgres bash -c "psql -c 'GRANT ALL PRIVILEGES ON DATABASE coobs TO coobs;'"
+	@sudo -iu postgres bash -c "psql -c 'ALTER USER coobs CREATEDB;'"
```

Comment:
Remember to also modify the `test` target so we can run test with just `make test`.